src/Form.jsx

```diff
@@ -137,7 +137,7 @@ function Form({
         activeStep
     ]);
 
-    function addRepeatedRow(fieldValuesArg = fieldValues) {
+    function addRepeatedRow({ values }) {
```

Comment:
This should default to `{ values = fieldValues }` since you call it with an empty object in the `onClick` handler